packages/scalecube-microservice/src/api/RouteOptions.ts

```diff
@@ -1,6 +1,15 @@
 import { LookUp, Message } from '.';
 
+/**
+ * The options that are used for the creation of the custom router
```

Comment:
/**
 * RouteOptions - The options that are used for the creation of the custom router
 *
 * @param lookUp - Functionality to find removeService by given critiria
 * 
 * @param message - Data that is used for executing the call of service method
 * 
 */